flixel/util/FlxStringUtil.hx

```diff
@@ -262,14 +262,8 @@ class FlxStringUtil
 	 */
 	public static function getDomain(url:String):String
```

Comment:
Oh, and I just realized that there is a problem with IPv4 addresses for hosts.  `http://127.0.0.1:3000/` would return `"0.1"` which is incorrect.  Although, that would have been a problem for the previous version of the function as well.  And of course, IPv6 breaks the parsing too.

At the end of the day, how much do we care about accuracy here?  There is a very simple and clear fix for this, but let me know if you think it is worth doing:

1. Define `getHost()` which returns the host section of the URL.  By having a new function here, we can define the behavior however we want without breaking anyone.
2. Update `getDomain()` to use `getHost()` and further break down the host into just the two-level domain, handling IP addresses and `localhost` as needed.

Side note... We should support `file://D:/folder/file.extension` as a local path, and make sure it does not get parsed as a remote host, just in case.